packages/core/docs/Dockerfile

```diff
@@ -4,9 +4,15 @@ WORKDIR /var/www
 
 COPY . .
 
-RUN npm install \
-   && npm run build
+RUN yarn install --network-concurrency 1 \
+    && yarn build:core \
+    && yarn build:ct:api-client \
+    && yarn cache clean --all
```

Comment:
Can you explain `--network-concurrency 1` and `yarn cache clean --all`?

Reply:
I think that was unnecessary. Removed it.